packages/gatsby-theme-newrelic/src/components/SearchInput.js

```diff
@@ -97,18 +98,18 @@ const SearchInput = forwardRef(
           }}
           css={css`
             width: 100%;
-            border: 1px solid var(--border-color);
+            border: 0px solid var(--border-color);
             border-radius: 4px;
-            background: var(--primary-background-color);
+            background: var(--color-neutrals-700);
```

Comment:
Does this work for both light and dark modes? This `SearchInput` is used globally, not just the header.